common/src/test/java/com/linecorp/line/auth/fido/fido2/common/server/bean/validation/AuthOptionRequestBeanValidationTest.java

```diff
@@ -40,9 +40,10 @@ void validateIncompleteRequestWithBlank() {
 
         final Set<ConstraintViolation<AuthOptionRequest>> constraintViolations = validator.validate(authOptionRequest);
 
-        assertThat(constraintViolations.size()).isEqualTo(1);
+        assertThat(constraintViolations).hasSize(1);
         constraintViolations.forEach(
```

Comment:
[containsOnly](https://www.javadoc.io/doc/org.assertj/assertj-core/latest/org/assertj/core/api/AbstractIterableAssert.html#containsOnly(ELEMENT...)) checks all the elements of a collection, so we can get rid of `forEach`

Reply:
Oh, thanks!